### PR TITLE
[Remove immediate=true usage core-infrastructure-4 part-2]Lps 168633 part 2

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-168633

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@slnn No regressions found.

File: modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/main/java/com/liferay/portal/lpkg/deployer/internal/BytesURLProtocolSupport.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 /**
  * @author Shuyang Zhou
  */
-@Component(immediate = true, service = BytesURLProtocolSupport.class)
+@Component(service = BytesURLProtocolSupport.class)
```
Comment (dantewang) [inline modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/main/java/com/liferay/portal/lpkg/deployer/internal/BytesURLProtocolSupport.java:42]: @slnn This class is not tracked or referenced or even used by anyone, but it registers a `URLStreamHandlerService` service when activating. I guess this service will be tracked by the OSGi framework, so we should keep this registration.

So I think we should keep this started by default.

**Comment by dantewang** [comment]
@slnn Please see my comment and resend. Thanks.

File: modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/main/java/com/liferay/portal/lpkg/deployer/internal/BytesURLProtocolSupport.java (hunk 1 above)
Comment (slnn) [reply modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/main/java/com/liferay/portal/lpkg/deployer/internal/BytesURLProtocolSupport.java:42]: Please see https://github.com/liferay-core-infra/liferay-portal/pull/2838

**Pull request closed without merging**